phoenix-core/src/main/java/org/apache/phoenix/parse/AlterIndexStatement.java

```diff
@@ -65,6 +67,10 @@ public boolean isAsync() {
         return async;
     }
 
+    public boolean IsRebuildAll() {
```

Comment:
Nit: Function names are better started with verbs and Rebuild fits fine. Suggest rename to RebuildAll() .. and make the member variable rebuildAll

Reply:
RebuildAll is misleading. It seems like it will do the actual rebuild. I will go with isRebuildAll